vars/utils.groovy

```diff
@@ -0,0 +1,8 @@
+def getCosaDir(params = [:]) {
+
```

Comment:
Minor: extra space here.

Reply:
Fixed.
Yes, I tested it in my centos namespace using the setup for cosa CI